### DISCUSSION TITLE
Check if player is in godmode?

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2334] [Q&A]
So i can get a peds' proofs but i don't get the god proof and i found no way to get the god proof.
Now is it possible to do this in a script tho?
`(src/views/esp/view_esp.cpp:88)`
```c++
if (ped_damage_bits & (uint32_t)eEntityProofs::GOD)
```

<br />
Also where can i find the most updated version of gta dumps? For example all scripts.
And is there a good tutorial on how i can find out locals?

**Comment by maybegreat48** [comment]
Duplicate of #2422

**Comment by pierrelasse** [comment]
Oh lol then i already asked that.
